nilearn/plotting/displays.py

```diff
@@ -1216,6 +1221,9 @@ def add_graph(self, adjacency_matrix, node_coords,
                               vmin=edge_vmin, vmax=edge_vmax,
                               **edge_kwargs)
 
+        if colorbar:
+            self._colorbar_show(ax, threshold=edge_threshold)
```

Comment:
This is very clever but a bit nasty. Basically you noticed that the `im` argument of _colorbar_show only needed a  `.cmap` and a `.norm` and you defined it appropriately on the GlassBrainAxes instance so that everything just works when you pass a GlassBrainAxes instance to _colorbar_show.

I am wondering whether the signature of the _colorbar_show function to have a `norm` and a `cmap` object and refactor the code accordingly. You will probably need to move the vmin, vmax computation here too rather than _add_lines so that you have both cmap and norm available here.